src/react-apps/applications/shared/src/components/AltinnInputField.tsx

```diff
@@ -3,6 +3,7 @@ import { withStyles } from '@material-ui/core/styles';
 import classNames from 'classnames';
 import * as React from 'react';
 import altinnTheme from '../theme/altinnStudioTheme';
+import AltinnButton from './AltinnButton';
 
 export interface IAltinnInputFieldComponentProvidedProps {
```

Comment:
Suggestion: Use VSCode sort function (ctrl+shift+p : sort), makes merging code easier in the future